plugins/modules/packaging/os/openbsd_pkg.py

```diff
@@ -166,6 +173,10 @@ def get_package_state(names, pkg_spec, module):
 # Function used to make sure a package is present.
 def package_present(names, pkg_spec, module):
     build = module.params['build']
+    snapshot = module.params['snapshot']
+
+    if snapshot is True and build is True:
+        module.fail_json(msg="the combination of build=%s and snapshot is not supported" % module.params['build'])
```

Comment:
```suggestion
        module.fail_json(msg="the combination of build=%s and snapshot is not supported" % module.params['build'])
```
make them mutually exclusive and add the note about this to the doc

Reply:
What do you mine with "make them mutually exclusive"? I have followed the same pattern as it is for `build` in `package_latest`.

Reply:
What do you mine with "make them mutually exclusive". I have followed the same pattern as it is in `package_latest`

Reply:
He means using the `mutually_exclusive` param to `AnsibleModule` (at the very beginning of the module's `main()` function). See:

https://github.com/ansible-collections/community.general/blob/main/plugins/modules/source_control/git_config.py#L165

as an example.

Reply:
I'm currently working on documenting these things, see here for the current version: https://github.com/ansible/ansible/blob/72d39015f7705ffe9b692318d5f34a0ea4f7ac1c/docs/docsite/rst/dev_guide/developing_program_flow_modules.rst#dependencies-between-module-options

Reply:
Since you are using `mutually_exclusive` now you no longer need to code this check yourself.

Reply:
Of course, thanks for the hints and review.